memgpt/server/rest_api/app.py

```diff
@@ -0,0 +1,169 @@
+import json
+import logging
+import secrets
+from pathlib import Path
+from typing import Optional
+
+import typer
+import uvicorn
+from fastapi import FastAPI
+from starlette.middleware.cors import CORSMiddleware
+
+from memgpt.server.constants import REST_DEFAULT_PORT
+
+# NOTE(charles): these are extra routes that are not part of v1 but we still need to mount to pass tests
+from memgpt.server.rest_api.auth.index import (
+    setup_auth_router,  # TODO: probably remove right?
+)
+from memgpt.server.rest_api.interface import StreamingServerInterface
+from memgpt.server.rest_api.routers.openai.assistants.assistants import (
+    router as openai_assistants_router,
+)
+from memgpt.server.rest_api.routers.openai.assistants.threads import (
+    router as openai_threads_router,
+)
+from memgpt.server.rest_api.routers.openai.chat_completions.chat_completions import (
+    router as openai_chat_completions_router,
+)
+
+# from memgpt.orm.utilities import get_db_session  # TODO(ethan) reenable once we merge ORM
```

Comment:
Note @norton120 @sarahwooders the only ORM specific code (I think?) that I had to comment out of this file